src/pybamm/parameters/bpx.py

```diff
@@ -222,6 +222,9 @@ def _bpx_to_param_dict(bpx: BPX) -> dict:
             domain.pre_name + "transport efficiency"
         ] ** (1.0 / 1.5)
 
+    def _get_activation_energy(var_name):
+        return pybamm_dict.get(var_name) or 0.0
```

Comment:
@rtimms I moved this function above the comment about the functions requiring more than 1 variable. I also renamed the function parameter since it shadowed the name from outer scope